### ISSUE TITLE
[ui] Implement UI to edit planet and profile info

### ISSUE THREAD

**Issue by emperorgurgeh** [emperorgurgeh/desolates-map#32]
A proper figma file will be given to whomever takes this task

<img width="871" alt="Screen Shot 2021-12-06 at 3 11 50 AM" src="https://user-images.githubusercontent.com/92417579/144810573-1202f6b5-d95d-4e47-bc9d-e4d14fbce5bb.png">

This task is just the UI implementation. Connecting it to the backend will be a separate task, to be done once Solana Sign-in is finished
